x/stakeibc/keeper/msg_server_restore_interchain_account.go

```diff
@@ -12,33 +12,29 @@ import (
 	"github.com/Stride-Labs/stride/v4/x/stakeibc/types"
 )
 
+// 6 function, 2 for verify record after exec, 4 for test (1 success, 3 for failure)
 func (k msgServer) RestoreInterchainAccount(goCtx context.Context, msg *types.MsgRestoreInterchainAccount) (*types.MsgRestoreInterchainAccountResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
 	hostZone, found := k.GetHostZone(ctx, msg.ChainId)
 	if !found {
-		k.Logger(ctx).Error(fmt.Sprintf("Host Zone not found: %s", msg.ChainId))
+		k.Logger(ctx).Error(fmt.Sprintf("Host Zone not found: %s", msg.ChainId)) //done
 		return nil, types.ErrInvalidHostZone
 	}
 
 	owner := types.FormatICAAccountOwner(msg.ChainId, msg.AccountType)
 
 	// only allow restoring an account if it already exists
-	portID, err := icatypes.NewControllerPortID(owner)
-	if err != nil {
-		errMsg := fmt.Sprintf("could not create portID for ICA controller account address: %s", owner)
-		k.Logger(ctx).Error(errMsg)
-		return nil, err
-	}
+	portID, _ := icatypes.NewControllerPortID(owner)
 	_, exists := k.ICAControllerKeeper.GetInterchainAccountAddress(ctx, hostZone.ConnectionId, portID)
 	if !exists {
-		errMsg := fmt.Sprintf("ICA controller account address not found: %s", owner)
+		errMsg := fmt.Sprintf("ICA controller account address not found: %s", owner) //done
 		k.Logger(ctx).Error(errMsg)
 		return nil, sdkerrors.Wrapf(types.ErrInvalidInterchainAccountAddress, errMsg)
 	}
 
 	if err := k.ICAControllerKeeper.RegisterInterchainAccount(ctx, hostZone.ConnectionId, owner); err != nil {
-		k.Logger(ctx).Error(fmt.Sprintf("unable to register %s account : %s", msg.AccountType.String(), err))
+		k.Logger(ctx).Error(fmt.Sprintf("unable to register %s account : %s", msg.AccountType.String(), err)) //adding (already have, but still: missing 2 cases)
```

Comment:
what is the mean of these comments?

Reply:
I should delete this after checking all conditions are fullfilled.

Reply:
u should delete this or anything else like this before you create a pr, it should be in your local not the remote man

Reply:
Sorry, I created this PR only for quick a review. I am not intending to merge this., I will remove the comments and create a PR like this to your branch.